csrankings-a.csv

```diff
@@ -1573,6 +1576,7 @@ Andy Cobley,University of Dundee,https://www.computing.dundee.ac.uk/about/staff/
 Andy Cockburn,University of Canterbury,http://www.cosc.canterbury.ac.nz/andrew.cockburn,HzupDnMAAAAJ
 Andy Crabtree,University of Nottingham,https://www.nottingham.ac.uk/computerscience/people/andy.crabtree,LX_P5GYAAAAJ
 Andy D. Perkins,Mississippi State University,http://web.cse.msstate.edu/%7Eperkins,NOSCHOLARPAGE
+Andy D. Pimentel,University of Amsterdam,https://staff.fnwi.uva.nl/a.d.pimentel/

```

Comment:
Third "no Google Scholar ID". Closing. Please fix all and check all DBLP names for aliases, and add them, then re-open.